Makefile

```diff
@@ -58,6 +58,7 @@ install-tools-linting: # Install tools that are used for linting.
 	go install -v github.com/kisielk/errcheck@v1.7.0
 	go install -v github.com/mgechev/revive@v1.4.0
 	go install -v honnef.co/go/tools/cmd/staticcheck@2024.1.1
+	go install golang.org/x/tools/cmd/deadcode@v0.16.1
```

Comment:
What is this for?

Reply:
To find dead code. There were some unused function. And the current static checkers we use did not find them.